pages/manage/productions/create.vue

```diff
@@ -14,6 +14,10 @@
             :key="suggestedEvent['@id']"
             :type="getEventType(suggestedEvent.terms)"
             :title="suggestedEvent.name[locale]"
+            :location-name="suggestedEvent.location.name[locale]"
```

Comment:
It's possible that there's no name for the given `locale`. In that case we should fall back on the `mainLanguage` of the `location` (the location is its own separate entity with its own mainLanguage, separate from the event - it can be that the event was originally entered in Dutch but the location in French).

So something like:
```suggestion
            :location-name="suggestedEvent.location.name[locale] ?? suggestedEvent.location.name[suggestedEvent.location.mainLanguage]"
```

For `title` we should probably do the same, but with the `mainLanguage` of `suggestedEvent`:
```
:title="suggestedEvent.name[locale] ?? suggestedEvent.name[suggestedEvent.mainLanguage]"
```